backtester/README.md

```diff
@@ -1,6 +1,6 @@
 # GoCryptoTrader Backtester: Backtester package
 
-<img src="https://github.com/gloriousCode/gocryptotrader/blob/backscratcher/backtester/common/backtester.png?raw=true" width="350px" height="350px" hspace="70">
+<img src="https://github.com/thrasher-corp/gocryptotrader/blob/master/backtester/common/backtester.png?raw=true" width="350px" height="350px" hspace="70">
```

Comment:
Just checking: wondering if we can make all these relative as well.  `<img src="../backtester/common/backtester.png?raw=true" width="350px" height="350px" hspace="70">`

Reply:
I tried, but given that everything has different depths I could not

Reply:
Ahh yeah, okay!

Reply:
Turns out I could make it relative and I just couldn't figure it out... But now its updated